sdk/spring/spring-cloud-azure-starter-monitor/src/main/java/com/azure/spring/cloud/autoconfigure/monitor/AzureSpringMonitorAutoConfiguration.java

```diff
@@ -34,33 +28,17 @@ class AzureSpringMonitorAutoConfiguration {
 
     private static final Logger LOG = LoggerFactory.getLogger(AzureSpringMonitorAutoConfiguration.class);
 
-    private static final Function<ConfigProperties, Map<String, String>> OTEL_DISABLE_CONFIG = configProperties -> {
-        Map<String, String> properties = new HashMap<>();
-        properties.put("otel.sdk.disabled", "true");
-        return properties;
-    };
+    private static final AutoConfigurationCustomizerProvider DISABLE_OTEL_CUSTOMER_PROVIDER = autoConfigurationCustomizer -> autoConfigurationCustomizer.addPropertiesCustomizer(configProperties -> new HashMap<String, String>(1) {{
+        put("otel.sdk.disabled", "true");
+    }});
```

Comment:
you could use (and static import) [`singletonMap()`](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#singletonMap-K-V-)